src/slack.coffee

```diff
@@ -242,4 +245,29 @@ class SlackBot extends Adapter
     channel = @client.getChannelGroupOrDMByName envelope.room
     channel.setTopic strings.join "\n"
 
+  customMessage: (data) =>
+    channel = @client.getChannelGroupOrDMByName data.message.envelope.room
+
+    msg = []
+    data.attachments = data.content if data.content and not data.attachments
+    data.attachments = [data.attachments] unless Array.isArray data.attachments
```

Comment:
It might be nicer to not mutate the `data` object. I don't know if clients that post objects to `slack-attachment` or `slack.attachment` ever expect the object to remain un-mutated, but it just seems unnecessary.